### ISSUE TITLE
Error loading Mod

### ISSUE THREAD

**Issue by livespectare** [TheAlchemist0033/alchemy#10]
Error loading Mod with the following message:

ModError: Failed to load and run script from /home/jorge/.minetest/mods/alchemy/init.lua:
/home/jorge/.minetest/mods/alchemy/mobs.lua:441: attempt to call local 'S' (a nil value)
stack traceback:
	/home/jorge/.minetest/mods/alchemy/mobs.lua:441: in main chunk
	[C]: in function 'dofile'
	/home/jorge/.minetest/mods/alchemy/init.lua:73: in main chunk

**Comment by TheAlchemist0033** [comment]
> Error loading Mod with the following message:
> 
> ModError: Failed to load and run script from /home/jorge/.minetest/mods/alchemy/init.lua:
> /home/jorge/.minetest/mods/alchemy/mobs.lua:441: attempt to call local 'S' (a nil value)
> stack traceback:
> 	/home/jorge/.minetest/mods/alchemy/mobs.lua:441: in main chunk
> 	[C]: in function 'dofile'
> 	/home/jorge/.minetest/mods/alchemy/init.lua:73: in main chunk

Ummmmmmmm its been a few years since I even looked at this, but im like 90% sure that its just because youre missing a dependency. Does the mobs mod still exist? it added dirt and stone monsters and a big dude that throws fireballs...

**Comment by livespectare** [comment]
Yes. Mod Mobs exists. I have it enabled on the game map I'm trying to load, but the error persists.

**Comment by TheAlchemist0033** [comment]
> Yes. Mod Mobs exists. I have it enabled on the game map I'm trying to load, but the error persists.

It might be a while since im so busy, but ill do my best to get to fixing it.

**Comment by AmericanHuston** [comment]
I'm also having the same issue. I think what's happening is mobs.intllib doesn't exist/isn't registered.

**Comment by AmericanHuston** [comment]
I've made the simple fix of commenting out the 5 instances when the local 'S' is used. 

In mobs.lua, lines 441-445
In earth_monster.lua, line 73 or 74.

**Comment by TheAlchemist0033** [comment]
This mod has not been supported since 2023. I don't play the game anymore, but I can take a look sometime within the next month or so and see if I can get things going. I'm certain other modpacks have changed and so they probably don't work with this version of the mod.

**Comment by AmericanHuston** [comment]
If you could explain what exactly its looking for, I might be able to fix it. What is it looking for when its assigning "S"?
